platform_secrets/service.py

```diff
@@ -66,6 +67,10 @@ async def add_secret(self, secret: Secret) -> None:
                 labels = {}
                 if secret.org_name:
                     labels[SECRET_API_ORG_LABEL] = secret.org_name
+                if secret.project_name == secret.owner:
+                    labels[USER_LABEL] = secret.owner.replace("/", "--")
```

Comment:
Unclear how we use it later, maybe you could elaborate? (maybe, the plan was to utilize it in `get_all_secrets` instead of _get_owner_from_secret_name)

Reply:
why wasn't this comment addressed?

Reply:
sorry, didn't notice your comment. Yes, I think the plan was to utilize it in `get_all_secrets` first but then I decided to refuse from this approach. This should be removed, will do.